src/generateTeam.js

```diff
@@ -0,0 +1,141 @@
+const Manager = require("../lib/Manager");
+const Engineer = require("../lib/Engineer");
+const Intern = require("../lib/Intern");
+
+// function that generates card for team (Manager and/or Engineer and/or Intern)
+function generateTeamCards(team) {
+  let cards = [];
+  // loop through team members and generate relevant card for each
+  for (let i = 0; i < team.length; i++) {
+    const teamMember = team[i];
+    switch (teamMember.getRole()) {
+      case "Manager":
+        const manager = new Manager(
+          teamMember.name,
+          teamMember.id,
+          teamMember.email,
+          teamMember.officeNumber
+        );
+        cards.push(generateManagerCard(manager));
```

Comment:
great use of the .push method alongside calling a very modular function directly inside of it!